recipe/meta.yaml

```diff
@@ -13,16 +13,13 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 200
```

Comment:
I see. Would you mind migrating that to the docs? Or making an issue to migrate it please?  Thanks!